### ISSUE TITLE
Negative records add little value

### ISSUE THREAD

**Issue by derek-cheong** [AdmiralGT/ttennis_ladder#46]
I think we should remove the following from the records page.  I can understand having positive records for people to aim to beat, but IMO these negative ones provide no incentives to play.
Longest Losing Streak (someone had a bad run, so what?  Nothing that player can do to change this.  If someone else gets close they might prefer to just stop playing rather than take this record)
Lowest Ranking Achieved (as above.  This will rarely change, and the current lowest ranked player can never change this)
Most Losses (why punish someone for playing a lot?)

**Comment by AdmiralGT** [comment]
They're not intended as incentives to play, they're intended as all-time records of stats that are already collected. Everyone has their own longest losing streak (worst run), lowest ranking achieved (rating range) and most losses (total losses). These purely provide a way to compare your current stats to the all-time stats without having to search through every player page.

**Comment by derek-cheong** [comment]
Fair point.  Is it worth considering removing the names for the negative records?  We can achieve the same thing without naming and shaming those in question.

**Comment by AdmiralGT** [comment]
Needs several CSS changes to the Records page (it's currently horrific so removing names from some records should be investigated at the same time).
